astropy/coordinates/angles.py

```diff
@@ -644,12 +650,20 @@ def _wrap_internal(self):
         # Convert the wrap angle and 360 degrees to the native unit of
         # this Angle, then do all the math on raw Numpy arrays rather
         # than Quantity objects for speed.
+        # TODO: This could probably be sped up as well.  u.degree.to(self.unit)
```

Comment:
Here, I would just leave a note that it might be possible to speed this up, both for `a360` and `wrap_angle`. I don't think `_wrap_internal` should be sufficiently common to worry too much about actually doing it, though, so improvements can hopefully made elsewhere.